package/yast2-ruby-bindings.spec

```diff
@@ -40,9 +40,9 @@ Requires:       rubygem(%{rb_default_ruby_abi}:fast_gettext) < 3.0
 BuildRequires:  ruby-devel
 Requires:       yast2-core >= 3.2.2
 BuildRequires:  yast2-core-devel >= 3.2.2
-# UI.SetApplicationTitle
-Requires:       yast2-ycp-ui-bindings       >= 4.2.2
-BuildRequires:  yast2-ycp-ui-bindings-devel >= 4.2.2
+# MenuBar widget
+Requires:       yast2-ycp-ui-bindings       >= 4.3.11
```

Comment:
I guess this should be 4.3.1.

Reply:
You are right.

I had spotted the wrong version number after I edited this. Updating.